skin.json

```diff
@@ -8,6 +8,7 @@
                   "scalingOption": "default",
                   "scalingPercentage": 20
     },
+    "bufferingSpinnerDelay": 750,
```

Comment:
This was 1250 in the skin PR, wasn't it? Assuming this is a deliberate change.

Reply:
Yes, this was Steve's suggestion